src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
```

Comment:
utility classes are anti-pattern

I would better rename this class to smth like `ConnectionAccessor` or `ConnectionProvider` or `ConnectionFactory` and inject it into the DAO

so instead of `ConnectionUtil.getConnection()` you would do `connectionProvider.getConnection()`